frontend/src/modules/table/table.types.tsx

```diff
@@ -56,7 +56,7 @@ type TableProps<Entry = TableEntry> = {
     focusProps?: any;
     focusFileUploadProps?: any;
     serverRequestProps?: ServerRequestProps;
-    serverFetch?: AsyncThunk<any, ServerRequestProps, any>;
+    serverFetch?: AsyncThunk<any, ServerRequestProps | undefined, any>;
```

Comment:
this is already optional, do we need to explicitly set it to undefined?

Reply:
Good call, this was a holdover from before and not necessary anymore.